custom-service-adapter/README.md

```diff
@@ -0,0 +1,19 @@
+# Custom Service Adatper with Knot.x
+
+In this tutorial we create a page: [http://localhost:8092/html/books.html](http://localhost:8092/html/books.html) books from database.
+Database access is not available via WebAPI thus we have 2 options:
+
+1. integrate with data source implementing WebAPI layer
```

Comment:
`Implement WebAPI layer to access database and then integrate with it using HTTP adapter`